cpu/lpc11u34/include/LPC11Uxx.h

```diff
@@ -662,5 +662,5 @@ typedef struct {
 #endif
 
 
-#endif  // __LPC11UXX_H__
+#endif  // LPC11UXX_H__
```

Comment:
Can you, while your at it change this to C-style comments (`/* LPC11UXX_H__ */`)?